pages/create/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { GetServerSideProps } from "next";
+import { auth } from '@/auth';
+import Layout from "@/components/Layout";
+import prisma from "@/lib/prisma";
+import React from "react";
+import CreateForm, { props } from "@/components/CreateForm";
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    const session = await auth(context);
+    const email = session?.user?.email
+    
+    if (!email || typeof email !== 'string') {
+        return {
+            redirect: {
+                destination: '/',
+                permanent: false
+            }
+        };
+    }
+    
+    const name =  await prisma.user.findUniqueOrThrow({
+        where: {
+            email: email
+        },
+        select: {
+            name: true,
+            id: true
+        }
+    })
+    return {
+        props: {
+            session: JSON.parse(JSON.stringify(session)),
+            name
+        }
+    }
+    
+};
+
+const createPostPage: React.FC<props> = (props) => {
+    return (
+        <Layout>
+            <CreateForm {...props} />
```

Comment:
Can you rename <CreateForm> to something more specific like <CreatePostForm> for clarity

Reply:
Doesn't look like you addressed this

Reply:
I changed it from "createFormPage" to "createPostPage", I should've realized it really similar so I changed it to "pageToCreateForms" now lol

Reply:
What I meant is rename the file `CreateForm` to `CreatePostForm`. Rename the class too.

Reply:
What class are you referring to?

Reply:
Changed name to `CreatePostForm`